packages/forms/src/UIForm/fields/Date/Time.component.js

```diff
@@ -0,0 +1,102 @@
+import React, { useState } from 'react';
+import PropTypes from 'prop-types';
+import { InputTimePicker } from '@talend/react-components/lib/DateTimePickers';
+
+import FieldTemplate from '../FieldTemplate';
+import { generateDescriptionId, generateErrorId } from '../../Message/generateId';
+
+function TimeWidget(props) {
+	const {
+		errorMessage,
+		id,
+		isValid,
+		options,
+		onChange,
+		onFinish,
+		schema,
+		value,
+		valueIsUpdating,
+	} = props;
+	const descriptionId = generateDescriptionId(id);
+	const errorId = generateErrorId(id);
+
+	const [state, setState] = useState({ errorMessage: '' });
+
+	function onBlur(event) {
+		onFinish(event, { schema });
+	}
+
+	function onTimeChange(event, { errorMessage: nextErrorMessage, textInput, time }) {
+		setState({ errorMessage: nextErrorMessage });
+		const payload = {
+			schema,
+			value: nextErrorMessage ? null : textInput,
+		};
+		onChange(event, payload);
+		if (!nextErrorMessage && time) {
+			onFinish(event, payload);
+		}
+	}
+
+	return (
+		<FieldTemplate
+			description={schema.description}
+			descriptionId={descriptionId}
+			errorId={errorId}
+			errorMessage={state.errorMessage || errorMessage}
+			id={id}
+			isValid={isValid}
+			label={schema.title}
+			required={schema.required}
+			valueIsUpdating={valueIsUpdating}
+		>
+			<InputTimePicker
+				id={id}
+				autoFocus={schema.autoFocus}
+				disabled={schema.disabled || valueIsUpdating}
+				readOnly={schema.readOnly}
+				value={value}
+				useSeconds={options.useSeconds}
+				onBlur={onBlur}
+				onChange={onTimeChange}
+			/>
+		</FieldTemplate>
+	);
+}
+
+TimeWidget.displayName = 'TimeWidget';
+
+TimeWidget.defaultProps = {
```

Comment:
I've heard that `defaultProps` will be deprecated. See my previous comment below!